tools/ffi_coverage/src/main.rs

```diff
@@ -184,6 +184,42 @@ fn collect_public_types(krate: &str) -> impl Iterator<Item = (Vec<String>, ast::
                 if let Some(item) = &krate.index.get(import.id.as_ref().unwrap()) {
                     recurse(item, krate, types, path, true, None);
                 } else if let Some(item) = &krate.paths.get(import.id.as_ref().unwrap()) {
+                    // This is a reexport of a hidden module, which works fine in HTML doc but
+                    // doesn't seem to be reachable anywhere in JSON.
```

Comment:
File an issue about this upstream somewhere?

CC @Manishearth

Reply:
I don't know if this is actually an upstream issue and I don't want to spend a day figuring it out.

Reply:
Yeah `doc(hidden)` is a mess, I would still recommend filing a bug with a non-reduced testcase, or mentioning it in zulip.